src/test/java/models/testSpanishCard.java

```diff
@@ -0,0 +1,32 @@
+package models;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+/**
+ * Created by Jason on 2/18/2016.
+ */
+public class testSpanishCard {
+    @Test
+    public void testGetSuit(){
+        Card c = new Card(4,Suit.Swords);
```

Comment:
The build is failing because you are calling Suit.Swords when it should be SpanishSuit.Swords

Check lines 11, 12, & 17